biomaj/bank.py

```diff
@@ -525,10 +527,36 @@ def save_session(self):
 
         self.bank = self.banks.find_one({'name': self.name})
 
+    def format_release(self, release, reverse=False):
+        """
+        Check the release does not contains any dot '.' character that should make MongoDB complain
+
+        :param release: release number to check
+        :type release: str
+        :param reverse: Do the reverse replacement
+        :type reverse: bool
+        :return: Release string with dot '.' replaced with character set in config (release.replace)
+        :rtype: str
+        """
+        if not self.config.get('release.replacedot'):
```

Comment:
I was thinking about something like this

Le mer. 27 avr. 2016 11:35, Emmanuel Quevillon <notifications@github.com> a
écrit :

> In biomaj/bank.py
> <https://github.com/genouest/biomaj/pull/44#discussion_r61229240>:
>
> > @@ -525,10 +527,36 @@ def save_session(self):
> >
> >          self.bank = self.banks.find_one({'name': self.name})
> >
> > +    def format_release(self, release, reverse=False):
> > +        """
> > +        Check the release does not contains any dot '.' character that should make MongoDB complain
> > +
> > +        :param release: release number to check
> > +        :type release: str
> > +        :param reverse: Do the reverse replacement
> > +        :type reverse: bool
> > +        :return: Release string with dot '.' replaced with character set in config (release.replace)
> > +        :rtype: str
> > +        """
> > +        if not self.config.get('release.replacedot'):
>
> Or store pending like sessions, inside an array of dict:
>
> {
>   "pending" : [
>                {"release": "6.10",  "id": 1461056695.108994},
>                {...}
>                ]
> }
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/genouest/biomaj/pull/44/files/d235f544d677e718229ab04b92229485fa50ca28#r61229240>
>